### ISSUE TITLE
Welp, This is broke | SOS!!!

### ISSUE THREAD

**Issue by Pythonix756** [Mithion/ArsMagica2#1548]
2 days ago I started a new world on my server and I started with Ars Magica, I got the essence and the book ready and got me an Arcane Compendium, I made an occulus and an inscription table. After that something happened which corrupted the world save and I fixed the save but everything was gone, So I had to rollback the server and when I did I had to do the basic things to get started but when I got the Arcane Compendium, it said what it says when you get it and My old entries were already unlock, And I got my occulus and clicked on it and it said I cant comprehend what I see inside the occulus, I go into creative and craft me a spell from the inscription table, Switch to survival and use the spell, My mana bar shows up, I see my mage level which is 1 and when I click the occulus it opens but Everything in it is locked, I got 3 skill points and everything is locked and I cant learn any shapes or anything and basically cant perform Ars Magica at all. Can someone please help me and tell me where is the Ars Magica data? Cuz I tried resetting the player data (My .dat file) and it still doesn't work, I deleted AM2PowerData in my world save and still doesn't work, Also I remove the mod and re-added it to remove everything related to it from the save still didn't work. Also I use a Thermos server with Multiverse on it if that could help, I use Ars Magica on the Multiverse world

**Comment by Pythonix756** [comment]
Oh also, It works in my main world but not my Multiverse world??? Why?

**Comment by Mithion** [comment]
No idea on the multiverse world (never really played with it TBH).
However, you should be able to fix your other issue by running the /respec command.

**Comment by Pythonix756** [comment]
Well Mithion thanks for the reply but the respec command never worked and I tell you it worked on Multiverse many times

**Comment by Pythonix756** [comment]
Ok this is annoying @Mithion , I created a new world, Or its new cuz I didnt perform any ars magica stuff in it and the arcane compendium still has my entries and the occulus isnt opening, Also I cant see my mana and burnout bar unless in creative

**Comment by Pythonix756** [comment]
@Mithion Alright this is getting depressing and desperate both at once, I really need help I cant play my modpack without Ars Magica

**Comment by Pythonix756** [comment]
![image](https://cloud.githubusercontent.com/assets/19330439/17083548/91fa9a96-5159-11e6-9fd5-dcc1ab09f336.png)
What you see right there is an image of a working occulus after grabbing me a Fire spell from the inscription table in creative and using it
![image](https://cloud.githubusercontent.com/assets/19330439/17083554/bc9c24cc-5159-11e6-800a-0692a77c3871.png)
And here is an image of the spell making but my issues is, I cant use the points I have in the occulus, Nothing is or can be learned and I try adding points using the orbs and it just doesnt work
Help needed

**Comment by Pythonix756** [comment]
Bump

**Comment by Pythonix756** [comment]
Well that does it, Worst magic mod I could ever find @Mithion

**Comment by Pythonix756** [comment]
Awful support @Mithion

**Comment by Mithion** [comment]
Couple things here:

First, you ping me incessantly when you're clearly on a different timezone than me.  Most of your messages come in at 3am for me (for example, your BM bump), and doing so is just rude.

Second, I stopped developing this mod almost a year ago.  It's open source so the community can work on it.  I already answered you once with a suggestion.  It didn't work unfortunately, but that isn't the point.  If I answer, it's because my schedule permits.  I am working 2 full time jobs right now to make ends meet.  This isn't my priority.

Third, I am under no obligation to support this mod or you.  Lashing out and calling it worst, etc. while simultaneously spamming the thread will get you nowhere.

Closed, locked, and good day.
